apps/dashboard/src/api/subscribers.ts

```diff
@@ -92,3 +94,34 @@ export const patchSubscriber = async ({
 
   return data;
 };
+
+export const getSubscriberPreferences = async ({
+  environment,
+  subscriberId,
+}: {
+  environment: IEnvironment;
+  subscriberId: string;
+}) => {
+  const { data } = await getV2<{ data: GetSubscriberPreferencesDto }>(`/subscribers/${subscriberId}/preferences`, {
```

Comment:
```suggestion
  return getV2<{ data: GetSubscriberPreferencesDto }>(`/subscribers/${subscriberId}/preferences`, {
```

Reply:
we use it like this in other apis and also the return type would change, breaking the hooks etc. 

If we want to change the return type then lets do it in all apis and hooks at once, for the sake of consistency.

Reply:
Ok, we will do it separately, although there is no need not to return the rest of the useQuery values, we need them in most cases anyway.